packages/shared/src/components/post/SharePostContent.tsx

```diff
@@ -69,7 +70,10 @@ function SharePostContent({
                 className="mt-5 btn-secondary w-fit"
                 href={post.sharedPost.permalink}
                 openNewTab={openNewTab}
-                onClick={onReadArticle}
+                title="Go to post"
+                rel="noopener"
+                onClick={(e) => e.stopPropagation()}
+                {...combinedClicks(onReadArticle)}
               />
             </div>
             <div className="block overflow-hidden ml-2 w-70 rounded-2xl cursor-pointer h-fit">
```

Comment:
The ticket also mentions the image being clickable, which seems to be missing below here 🙏

Reply:
I thought that mean't the image on the post page needed to link which is does. So in this panel, the image, button and title should link to the actual article and not our post page?
![Screenshot 2023-08-02 at 11 13 52 am](https://github.com/dailydotdev/apps/assets/42202149/783e543f-f587-4485-8bf6-f5173c540c73)

Reply:
Yeah there basically the whole card is a link already right?
Just the readpost should do something else.

My part is indeed about the normal article modal, but seems that works as intended.